### DISCUSSION TITLE
Can't get the Student Pack !

### DISCUSSION THREAD

**Discussion by GutsDCEO** [community/community#145806] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hello , 
I couldn't get the student pack , it got stuck in here after I press "Continue" without asking me for any school documentations . If you could help me out with this it will be much appreciated.
Best regards. 
![Capture](https://github.com/user-attachments/assets/c533f0d7-d82a-4a96-9561-b2a77ad1a218)

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @GutsDCEO , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by GutsDCEO** [reply]
Can I get a reply from you ? How can I get my student pack as a student as fast as possible ? I need it for my academic projects .
